### PR TITLE
Upgrade to v6 unstable 1378

### PR DESCRIPTION
**Pull request by colin-higgins**
@kbaley Please review

cc / @DavidBoike shouldn't really be any conflicts

### CODE REVIEW

File: src/NServiceBus.RavenDB/SessionManagement/OpenAsyncSessionBehavior.cs [hunk 1]
```diff
@@ -55,13 +55,13 @@ public Registration()
 
     class RavenAsyncSessionProvider : IAsyncSessionProvider
     {
-        readonly BehaviorContext context;
+        readonly IBehaviorContext context;
```
Comment (colin-higgins) [inline src/NServiceBus.RavenDB/SessionManagement/OpenAsyncSessionBehavior.cs]: @kbaley This is the relevant change

**Comment by kbaley** [comment]
:shipit:

**Comment by SimonCropp** [comment]
@colin-higgins @kbaley either of u interested in having a go the updating the ravendb samples https://github.com/Particular/docs.particular.net/issues/1017 ?

**Comment by kbaley** [comment]
@SimonCropp We've already started. I'll assign myself to the issue. /cc @colin-higgins

**Pull request merged**